### ISSUE TITLE
all streams selected using RCS

### ISSUE THREAD

**Issue by aaronjnewman** [labstreaminglayer/App-LabRecorder#111]
Hi

I am launching LabRecorder from a PsychoPy script, then starting recording automatically. Everything works fine, except that when the recording starts, **ALL available LSL streams are automatically selected and saved in the recording**. This occurs even though I have defined my RequiredStreams in LabRecorder.cfg, and only those streams are selected when LabRecorder is launched. It's only when recording starts that all available LSL streams are selected.

This is problematic because my lab has 3 testing rooms on the same VLAN, so if two experiments are running at the same time, the data from 2-3 experiments will get recorded in the both files. 

Is there a way to override this behavior? 

The commands I'm running in the PsychoPy script are:

```python
pid = Popen(['./LabRecorder/LabRecorder.exe'], stdin=PIPE)
labrecorder = socket.create_connection(("localhost", 22345))
labrecorder.sendall(b"filename {task:story%s} {session:00%s} {participant:%s} {modality:%s}\n" % (expt_mode.encode(), session.encode(), subj_id.encode(), modality.encode()))
labrecorder.sendall(b"start\n")
```

Thanks in advance

**Comment by aaronjnewman** [comment]
actually I see now this is addressed in #102, but that modification is still in progress. If nothing, else, consider this a +1 for getting that modification implemented - the current behavior makes RCS LabRecorder effectively unusable in my lab.

**Comment by cboulay** [comment]
@aaronjnewman , if you are willing and able, it'd be great if you could fork off #102 and make the changes I requested in that PR, then make a new PR from that.
I'm sorry but I don't have time to do it myself.
